lib/tools/pens/ControlBoundsPen.js

```diff
@@ -0,0 +1,77 @@
+define(
+    [
+        'ufojs/tools/pens/BasePen'
+      , 'ufojs/tools/misc/arrayTools'
+    ],
+    function(
+        Parent
+      , arrayTools
+) {
+    "use strict";
+
```

Comment:
to make it better visible what our dependencies are:
```
    var updateBounds = arrayTools.updateBounds ;
```
and later use updateBounds directly